tests/tests.py

```
@@ -1039,15 +1208,61 @@ def test_scopes_submitted_to_cluster(mocker):
+        dpath("test_group_jobs_resources"),
+        cluster="./qsub",
+        cores=6,
+        resources={"mem_mb": 60000},
+        max_threads=1,
+        overwrite_resource_scopes={"fake_res": "excluded"},
+        group_components={0: 5},
+        default_resources=Resources.parse(
+            ["mem_mb=0"], defaults="full", allow_expressions=True
+        ),
+    )
+    assert False
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)